pkg/gateway/model_build_targetgroup_test.go

```diff
@@ -145,7 +145,10 @@ func Test_TGModelByServicexportBuild(t *testing.T) {
 			}
 
 			ds := latticestore.NewLatticeDataStore()
-
+			if !tt.svcExport.DeletionTimestamp.IsZero() {
+				tgName := latticestore.TargetGroupName(tt.svcExport.Name, tt.svcExport.Namespace)
+				ds.AddTargetGroup(tgName, "vpc-123456789", "123456789", "tg-123", false, "")
+			}
```

Comment:
when test the ServiceExport deletion, the latticestore should already have a entry for it, so that the built tg.Spec.LatticeID will have this tgId, and it will be pass to the target_group_manage to delte this tg